authors/apps/authentication/views.py

```diff
@@ -106,3 +107,45 @@ def get(self, request):
 
     def sendResponse(self, message, status=status.HTTP_400_BAD_REQUEST):
         return Response({"message": message}, status)
+
+
+class PasswordResetAPIView(generics.GenericAPIView):
+    # Allow any user (authenticated or not) to hit this endpoint.
+    permission_classes = (AllowAny,)
+
+    def post(self, request):
+        try:
+            get_object_or_404(User, email=request.data['email'])
+            message = [
+                request,
+                "reset-password/change",
+                str((jwt.encode({"email": request.data['email']}, settings.SECRET_KEY)).decode('utf-8')),
+                "Reset Password",
+                "requested for password reset.",
+                request.data['email']
+            ]
+            Utilities.email_renderer(message)
+            return Response({"message": "Please check your email for the reset password link."}, status.HTTP_200_OK)
+        except KeyError:
+            raise exceptions.ValidationError("Email is required in-order to reset password.")
```

Comment:
thanks, @BagzieGracious ,we should let all messages we send to the user be as descriptive and correct as possible, we could change ```in-order``` to ```in order```.

Reply:
i have worked on it